deployment/supervisord_initd.sh

```diff
@@ -44,6 +44,12 @@ ulimit -n 96000
 
 RETVAL=0
 
+# source default env vars
```

Comment:
I didn't want to involve host environment in the application deployment. We can define other sorts of variables which are only set during image build time and they will be available to all processes.